bedrock/firefox/templates/firefox/all.html

```diff
@@ -30,7 +30,7 @@
   sky firefox-all-{{ channel }} {% if channel == 'alpha' %}blueprint{% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
FWIW, my gut feels the approach here (shared/cached bundle and page-specific bundle) is the right thing to do, but my gut cannot produce any supporting evidence.